ps2/Makefile

```diff
@@ -0,0 +1,69 @@
+DEBUG              = 0
+MUTE_WARNINGS      = 1
+PS2_IP = 192.168.1.150
```

Comment:
I prefer keeping this out of the `Makefile` and use an env var + a comment in the `Makefile` (see the other comment in the review).
```suggestion
```